source/code/source/playfab/PlayFabEventBuffer.cpp

```diff
@@ -98,7 +98,7 @@ namespace PlayFab
         // create an event packet, set it to the tail->next and move the tail
 
         uint64_t currentEventIndex = eventIndex->fetch_add(1, std::memory_order_relaxed);
-        PlayFabEventPacket* event = CreateEventPacket(reinterpret_cast<uint8_t*>(eventStart), currentEventIndex, std::move(request));
+        PlayFabEventPacket* event = CreateEventPacket(reinterpret_cast<uint8_t*>(eventStart), currentEventIndex, request);
```

Comment:
>request [](start = 113, length = 7)

this std::move is not excessive and properly transfers ownership to CreateEventPacket. This does not need to be removed